### PR TITLE
Add redfish processor architecture to Redfish CPU inventory

### LINKED ISSUE #6863: Get processor architecture from Redfish CPU inventory
### Summary

I'm trying to get the processor architecture and this is a field that is not present in the result of redfish_info GetCpuInventory command.

### Issue Type

Feature Idea

### Component Name

redfish_info

### Additional Information

```yaml (paste below)

```

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by pyfontan**
##### SUMMARY
This update adds ProcessorArchitecture to redfish CPU inventory 

Fixes #6863

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
community.general.redfish_info

### CODE REVIEW

**Comment by mraineri** [comment]
Other than the missing changelog fragment, this looks good to me!

**Comment by felixfontein** [comment]
Thanks for your contribution! You can find infos on changelog fragments here: https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment

**Comment by pyfontan** [comment]
Thank you, 

I hope that d30345739d9cd61685406e54e64495d320f66a45 will be ok.

File: changelogs/fragments/6864-redfish-utils-fix-for-processorarchitecture-in-cpu-inventory.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - redfish_info - Adds ProcessorArchitecture to CPU inventory (https://github.com/ansible-collections/community.general/pull/6864)
```
Comment (felixfontein) [inline changelogs/fragments/6864-redfish-utils-fix-for-processorarchitecture-in-cpu-inventory.yaml:2]:
```suggestion
  - redfish_info - adds ``ProcessorArchitecture`` to CPU inventory (https://github.com/ansible-collections/community.general/pull/6864).
```

**Comment by pyfontan** [comment]
@felixfontein Thank you for your suggestion

**Comment by felixfontein** [comment]
@pyfontan thanks for your contribution!
@mraineri thanks for reviewing!

**Comment by felixfontein** [comment]
Please open a PR against `main` the next time. I only now noticed it was against `stable-7` and not `main`... I cherry-picked the commit to `main` for this time.

**Comment by pyfontan** [comment]
> Please open a PR against `main` the next time. I only now noticed it was against `stable-7` and not `main`... I cherry-picked the commit to `main` for this time.

@felixfontein Sure, i will. 
Sorry for the inconvenience.

**Comment by felixfontein** [comment]
@pyfontan no worries, it's also my fault - I totally overlooked it :D And thanks again for contributing!

**Pull request merged**